GVFS/GVFS.UnitTests/Service/RepoRegistryTests.cs

```diff
@@ -7,20 +8,22 @@
 using System.Collections.Generic;
 using System.IO;
 using System.Linq;
+using System.Text;
 
-namespace GVFS.UnitTests.Windows.Service
+namespace GVFS.UnitTests.Service
 {
     [TestFixture]
     public class RepoRegistryTests
     {
         [TestCase]
         public void TryRegisterRepo_EmptyRegistry()
         {
-            string dataLocation = @"mock:\registryDataFolder";
+            string dataLocation = Path.Combine("mock:", "registryDataFolder");
+
             MockFileSystem fileSystem = new MockFileSystem(new MockDirectory(dataLocation, null, null));
-            RepoRegistry registry = new RepoRegistry(new MockTracer(), fileSystem, dataLocation);
+            RepoRegistry registry = new RepoRegistry(new MockTracer(), fileSystem, dataLocation, new GVFSMountProcess(new MockTracer()));

```

Comment:
This should be an `IRepoMounter` mock not `GVFSMountProcess` right?  This goes for all the places that you are creating a `new RepoRegistry`.